Graphics

secretrobotron opened the pull request:
This should get us going in the right direction. I set up some listeners for scene creation so that the graphics service can render things with active cameras in them.

Obviously, it doesn't do a whole lot yet but spin, but it's a start.

secretrobotron committed:
1. Graphics service starts up, is aware of scenes, and starts a render task.
2. Render task prepares for scenes->cameras->models, but doesn't draw yet.
3. Tests for the above.

Commit:
Merge pull request #65 from alankligman/work

Work

modeswitch commented on src/core/scene.js:
```diff
@@ -14,6 +14,13 @@ define( function ( require ) {
 
     return function( engine ) {
 
+        var __sceneAdded = new Event();
```
Define this in gladius-src.js on the engine directly. This function may be invoked multiple times.

secretrobotron committed:
1. Merge remote-tracking branch 'upstream/develop' into graphics
2. event.bind changed to event.subscribe

Conflicts:
	src/graphics/service.js

secretrobotron committed:
In the middle of changing around the renderer to use postMessage.

secretrobotron committed:
1. Camera component fleshed out a little.
2. Removed task from graphics service.
3. Added rAF for future use to graphics service.
4. Test for Model graphics component.
5. Fixed test for graphics service to include mesh, model, & service.
6. Fixed model and mesh to use correct require paths.
7. Fixed some typos in the graphics service render function.
8. Added CubicVR.renderObject to graphics service render function with reference to Transform component on model.
9. Added _cvr object and property to camera to match mesh.
10. Added mesh property to model to make it accessible.

secretrobotron commented:
This is useless now. CLOSING.
